.wordlist.txt

```diff
@@ -1,219 +1,230 @@
-MichaIng
-Joulinar
-StephanStS
-Meveric
-dietpi
-DietPi
-mkdocs
-MkDocs
-SBC
-SBCs
-RPi
-RPis
-Raspbian
-Hardkernel
-Odroid
-Odroids
+AirPlay
+Airsonic
+AIY
 Allo
-ASUS
-FriendlyARM
-NanoPi
-NanoPC
-ZeroPi
-Pinebook
+Amiberry
+Ampache
+analytics
+APIs
 Armbian
-systemd
-systemctl
-journald
-journalctl
+ARMv
+ASUS
+Audiophonics
+BaiKal
+balancer
+balenaEtcher
+Bazarr
+Blynk
+CalDAV
+CardDAV
+CAVA
+Certbot
+CIFS
+CLI
+Clonezilla
+Cloudflare
+CloudPrint
+Cloudshell
 containerd
-VM
-VirtualBox
-VMware
-NUC
-macOS
-FOSS
-UUID
-UEFI
-GPT
+CORS
+CouchPotato
+COVID
+cron
+CSI
 CSM
-IoT
-FHS
-SSD
-eMMC
-GPIO
-UART
-PCIe
-LPDDR
-EEPROM
-SSID
-TCP
-ICMP
-HTTPS
-SSL
+Cuberite
+DarkBetter
+DarkIce
+dietpi
+DIY
+DLNA
 DNS
-VPN
-VNC
-SCP
-SFTP
-RDP
-CORS
-CIFS
-SMB
-NTFS
-exFAT
-GlusterFS
-DXX
-OpenGL
-OpenGLES
-DRM
+Domoticz
 DPMS
-SPI
-CSI
+DRM
+Dropbear
 DSI
-LEDs
-DIY
-UI
-CLI
-APIs
-GiB
-MiB
-KiB
-Gbit
-Mbit
-Kbit
-MTA
-balenaEtcher
-cron
-balancer
-NFS
-Kodi
-Plex
+DXX
+EEPROM
 Emby
-Tautulli
-Signalyst
-Roon
-MotionEye
-MPD
-ympd
-myMPD
-CAVA
-Mopidy
-Airsonic
-Logitech
-LMS
-SqueezeLite
-DLNA
-MiniDLNA
-ReadyMedia
-UPnP
-DarkIce
+eMMC
+EmonPi
+exFAT
+FHS
+FOSS
+FreshRSS
+FriendlyARM
+Gbit
+GiB
+Gitea
+github
```

Comment:
Actually I wanted to avoid the need to add every word in lower case another time. Those errors are coming from links where generally spell checking should not be done, while in text passages, "GitHub" should always be written with this capitalisation, so marking "github" as mistake there is reasonable.